### DISCUSSION TITLE
How to export/import Rules Engine configuration from dashboard?

### DISCUSSION THREAD

**Discussion by matthewGEA** [emqx/emqx#4618] [Q&A] [answered]
per this post https://github.com/emqx/emqx-rule-engine/issues/139#issuecomment-653300689 after v4.1.0, there is supposed to be a way to export the rules engine configuration.  I am running v4.3-rc.3 and do not see any export/import options in the dashboard.

Could someone suggest where this option is?

**Comment by matthewGEA** [comment, marked as answer]
Per this link, the dashboard option to perform a backup/restore appears to be Enterprise only:
https://docs.emqx.io/en/broker/v4.3/advanced/data-import-and-export.html

> EMQ X Broker provides Command Line Interface, [HTTP API](./http-api.md#endpoint-data- import-and-export) and Dashboard's visual interface (Enterprise Edition) for data import and export functions.

**Comment by philippdzm** [comment]
How is this done in v5?
